packages/pds/src/repo/blobs.ts

```diff
@@ -9,28 +9,20 @@ import { BlobRef, PreparedWrites } from './types'
 import { Blob as BlobTable } from '../db/tables/blob'
 import * as img from '../image'
 import { sha256Stream } from '@atproto/crypto'
+import { fileTypeFromStream } from 'file-type'
 
 export const addUntetheredBlob = async (
   dbTxn: Database,
   blobstore: BlobStore,
   mimeType: string,
   blobStream: stream.Readable,
 ): Promise<CID> => {
-  const maybeGetImgInfo = async (
-    readable: stream.Readable,
-  ): Promise<img.ImageInfo | null> => {
-    if (mimeType.startsWith('image')) {
-      return img.getInfo(readable)
-    } else {
-      return null
-    }
-  }
-
-  const [tempKey, size, sha256, imgInfo] = await Promise.all([
+  const [tempKey, size, sha256, imgInfo, fileType] = await Promise.all([
     blobstore.putTemp(cloneStream(blobStream)),
     streamSize(cloneStream(blobStream)),
     sha256Stream(cloneStream(blobStream)),
-    maybeGetImgInfo(cloneStream(blobStream)),
+    img.maybeGetInfo(cloneStream(blobStream)),
+    fileTypeFromStream(blobStream),
```

Comment:
so this is actually the only way i could get it to work 🙃 
cloning each stream except for the file type one 

i'm not super pleased with this, but it does work & im honestly not sure what else to do right now 😅